esmvalcore/_recipe/recipe.py

```diff
@@ -212,9 +212,6 @@ def _get_default_settings(dataset):
 
     settings = {}
 
-    # Configure (deprecated, remove for v2.10.0) load callback
```

Comment:
I *think* this can be safely removed, since the `load` step is handled in `Dataset.load`. For example, `fix_metadata` settings are also **not** considered here.

Reply:
NVM, it is used...will revert